tests/evaluation.py

```diff
@@ -185,6 +186,32 @@ def test_duckling_patching():
     ]]
     assert patch_duckling_entities(entities) == patched
 
+def test_prepare_data():
+    td = load_data('data/examples/rasa/demo-rasa.json')
+    clean_data = prepare_data(td, 0)
+    unique_intents = sorted(set([i.data["intent"] for i in clean_data]))
+    assert(unique_intents == ['affirm', 'goodbye', 'greet', 'restaurant_search'])
+
+    clean_data = prepare_data(td, 10)
+    unique_intents = sorted(set([i.data["intent"] for i in clean_data]))
+    assert(unique_intents == ['affirm', 'restaurant_search'])
+
+def test_run_cv_evaluation():
+    import numpy as np
+    td = load_data('data/examples/rasa/demo-rasa.json')
+    n_folds = 3
+    nlu_config = RasaNLUConfig("sample_configs/config_defaults.json")
+
+    np.seed(2018)
+    results = run_cv_evaluation(td, n_folds, nlu_config)
+
+    rel_tol=1e-09
+    abs_tol=0.01
+
+    a = np.mean(results["accuracy"])
+    b = 0.65 # expected result
+    assert(abs(a-b) <= max(rel_tol * max(abs(a), abs(b)), abs_tol))
```

Comment:
can we use numpy here instead? https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.testing.assert_approx_equal.html